lib/Core/Executor.cpp

```diff
@@ -1104,9 +1131,17 @@ void Executor::executeGetValue(ExecutionState &state,
     std::set< ref<Expr> > values;
     for (std::vector<SeedInfo>::iterator siit = it->second.begin(), 
```

Comment:
Instead of keeping and introducing new unreadable iterator-based loops I'd prefer range-based for loops from C++11.

Reply:
Not my modification :)

Reply:
I know, but there are more iterators in the new code and I suggested to update the old sections too because the PR changes a great deal of it anyway.

Reply:
Very nice idea, but it does not belong to this PR.

Reply:
I'm not implying you should change every iterator loop in KLEE, just the ones in/around the changed sections.

Reply:
I perfectly understood. Still, I will not change code that does not belong to this PR. There are enough changes and I do not want to alter code that does not directly refer to array optimizations.